app/src/features/surveys/observations/sampling-sites/components/SamplingSiteMapControl.tsx

```diff
@@ -120,8 +120,8 @@ const SamplingSiteMapControl = (props: ISamplingSiteMapControlProps) => {
           <Typography component="legend" data-testid="funding-source-list-found">
             Site Location Preview &zwnj;
             <Typography component="span" color="textSecondary" fontWeight="400">
-              {formikProps.values.survey_sample_sites.length > 0
-                ? `(${formikProps.values.survey_sample_sites.length} locations detected)`
```

Comment:
We should probably use the pluralize util function to avoid rendering "1 locations detected"